### PR TITLE
Add OpenCL support for cracking 1Password Cloud Keychains

### PR DESCRIPTION
**Pull request by kholia**
This fixes https://github.com/magnumripper/JohnTheRipper/issues/3413.

### CODE REVIEW

File: src/opencl_cloudkeychain_fmt_plug.c [hunk 1]
```diff
@@ -0,0 +1,434 @@
...
+                cs.hmacdata[i] = atoi16[ARCH_INDEX(p[i * 2])] * 16
+                        + atoi16[ARCH_INDEX(p[i * 2 + 1])];
+
+	// we append the count and EOM here, one time <- salt hack required by our opencl kernel
+	memcpy(&cs.salt[cs.saltlen], "\x0\x0\x0\x1\x80", 5);
+	cs.saltlen = cs.saltlen + 5; // we include the x80 byte in our saltlen, but the .cl kernel knows to reduce saltlen by 1
+
+        MEM_FREE(keeptr);
+        return (void *)&cs;
+}
```
Comment (magnumripper) [inline src/opencl_cloudkeychain_fmt_plug.c]: Something is amiss with indentation above. Looks like spaces instead of tabs in most of the function?

Comment (kholia) [reply src/opencl_cloudkeychain_fmt_plug.c]: Yep, good catch. Fixed now.

**Comment by kholia** [comment]
TraviCI has been stuck on `Waiting for status to be reported` since 8 hours. Googling this situation suggests opening a support ticket with GitHub.

@magnumripper Can you please look into this? Thanks!

**Comment by magnumripper** [comment]
Oh. I know exactly what is wrong... I'll force merge this and try to fix the Travis problem.

**Comment by magnumripper** [comment]
@kholia @claudioandre-br 
The problem was I inadvertently deleted the deprecated github service triggering Travis and apparently there were no way to fix that. We have now migrated to travis-ci.com (as opposed to travis-ci.org) and things should be well again.

**Pull request merged**